mail/models.py

```diff
@@ -34,3 +34,20 @@ class AutomaticEmail(TimestampedModel):
     def __str__(self):
         """String representation of AutomaticEmail"""
         return "AutomaticEmail sender={}, subject={}".format(self.sender_name, self.email_subject)
+
+
+class SentAutomaticEmail(TimestampedModel):
+    """
+    Keeps track of automatic emails which were sent to particular users
+    """
+    user = models.ForeignKey(User, null=False, on_delete=models.CASCADE)
+    automatic_email = models.ForeignKey(AutomaticEmail, null=False, on_delete=models.CASCADE)
```

Comment:
It will be required to make this explicit for Django 2.0: https://docs.djangoproject.com/en/1.10/ref/models/fields/#django.db.models.ForeignKey

Regarding `CASCADE` vs `SET_NULL`, I figured there wasn't much point keeping `SentAutomaticEmail` here if one of the two fields were null